force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Salesforce Flows: Flow without access restrictions
#### Severity
Warning

#### Finding
This flow doesn't define any access restriction. Permissions to run certain flows should be defined either at the profile or at the permission set level.

#### Why is this a problem?
As a best practice, flows should be restricted to specific users. Omitting restrictions gives all users the possibility to invoke a flow, which is typically undesirable.